flow-bridge/src/main/java/org/reactivestreams/ReactiveStreamsFlowBridge.java

```diff
@@ -34,12 +34,12 @@ private ReactiveStreamsFlowBridge() {
         if (flowPublisher == null) {
             throw new NullPointerException("flowPublisher");
         }
-        if (flowPublisher instanceof org.reactivestreams.Publisher) {
-            return (org.reactivestreams.Publisher<T>)flowPublisher;
-        }
         if (flowPublisher instanceof FlowPublisherFromReactive) {
```

Comment:
By the way, before release we really should revisit naming of the classes `FlowPublisherFromReactive ` (specifically Reactive => ReactiveStreams or similar)

Reply:
Yeah, let's do the naming discussion separately.